### ISSUE TITLE
lost the ability to pass `-f json` for json output

### ISSUE THREAD

**Issue by wenottingham** [ansible/tower-cli#272]
tower-cli 3.0.3:
```
[notting@nostromo: ~/prog/tower-populate]$ tower-cli job_template get 13 --format json
{
  "id": 13, 
  "type": "job_template", 
  "url": "/api/v1/job_templates/13/", 
  "related": {
...
```

tower-cli 3.1.1:
```
[notting@nostromo: ~/prog/tower-populate]$ tower-cli job_template get 13 --format json
Error: no such option: --format
```

**Comment by sebastiangraf** [comment]
We experience the same, that's an major issue not visible in the release notes and at the moment blocking us massively.

**Comment by jangsutsr** [comment]
@sebastiangraf @wenottingham Thx for the feedback. That's a issue introduced by 3.1.1 and so sorry about the problem caused.

We've patched that up and it will ship with 3.1.2. For now two workarounds are available:
1. Directly set up config file, please see instructions [here](https://github.com/ansible/tower-cli/blob/master/docs/CONFIG_CMD_OPTIONS.md).
2. Use master branch with patch added.

**Comment by AlanCoding** [comment]
The fix has been shipped in release 3.1.2, which is on PyPI now.

**Comment by sebastiangraf** [comment]
thanks a lot!
